From ff4000bbf5d27a54e9ae01a7f89f35fe54f26566 Mon Sep 17 00:00:00 2001
Message-ID: <ff4000bbf5d27a54e9ae01a7f89f35fe54f26566.1719691458.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1719691458.git.sharlatanus@gmail.com>
References: <cover.1719691458.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sat, 13 Apr 2024 23:36:55 +0100
Subject: [PATCH 01/26] gnu: Add packages/golang-math module.

* gnu/packages/golang-maths.scm: New file.
* gnu/local.mk: Add gnu/packages/golang-maths.scm to the scope.

Change-Id: Iffa3f7735e2155ecdd46f5cdeda9143c9c823322
---
 gnu/local.mk                  |  1 +
 gnu/packages/golang-maths.scm | 43 +++++++++++++++++++++++++++++++++++
 2 files changed, 44 insertions(+)
 create mode 100644 gnu/packages/golang-maths.scm

diff --git a/gnu/local.mk b/gnu/local.mk
index 7528a6bfee..3e4ca7ceb8 100644
--- a/gnu/local.mk
+++ b/gnu/local.mk
@@ -320,6 +320,7 @@ GNU_SYSTEM_MODULES =				\
   %D%/packages/golang-check.scm		\
   %D%/packages/golang-compression.scm		\
   %D%/packages/golang-crypto.scm	      	\
+  %D%/packages/golang-maths.scm		\
   %D%/packages/golang-web.scm	        	\
   %D%/packages/golang-xyz.scm	        	\
   %D%/packages/gperf.scm			\
diff --git a/gnu/packages/golang-maths.scm b/gnu/packages/golang-maths.scm
new file mode 100644
index 0000000000..f7fc58ae2f
--- /dev/null
+++ b/gnu/packages/golang-maths.scm
@@ -0,0 +1,43 @@
+;;; GNU Guix --- Functional package management for GNU
+;;;
+;;; This file is part of GNU Guix.
+;;;
+;;; GNU Guix is free software; you can redistribute it and/or modify it
+;;; under the terms of the GNU General Public License as published by
+;;; the Free Software Foundation; either version 3 of the License, or (at
+;;; your option) any later version.
+;;;
+;;; GNU Guix is distributed in the hope that it will be useful, but
+;;; WITHOUT ANY WARRANTY; without even the implied warranty of
+;;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;;; GNU General Public License for more details.
+;;;
+;;; You should have received a copy of the GNU General Public License
+;;; along with GNU Guix.  If not, see <http://www.gnu.org/licenses/>.
+
+(define-module (gnu packages golang-maths)
+  #:use-module ((guix licenses) #:prefix license:)
+  #:use-module (guix build-system go)
+  #:use-module (guix gexp)
+  #:use-module (guix git-download)
+  #:use-module (guix packages)
+  #:use-module (gnu packages))
+
+;;; Commentary:
+;;;
+;;; Please: Try to add new module packages in alphabetic order.
+;;;
+;;; Code:
+
+;;;
+;;; Libraries:
+;;;
+
+;;;
+;;; Executables:
+;;;
+
+;;;
+;;; Avoid adding new packages to the end of this file. To reduce the chances
+;;; of a merge conflict, place them above in alphabetic order.
+;;;
-- 
2.41.0

